### PR TITLE
91 add localization to all texts and plural forms

### PR DESCRIPTION
**Pull request by drucelweisse**
# Description

this pr add localization to A11yDescription

## Changes
- added Localization enum based on Localizable files (should it be generated?)
- changed A11yDescription voiceOverText to insert localized strings

## TODO:
- [x] add correct voiceOverText generation
- [x] configure to run tests with different locales (probably config in Test Plan)
- [x] fix tests for A11yDescription

### CODE REVIEW

**Comment by drucelweisse** [comment]
Should we specify in document which language we cover?

**Comment by akaDuality** [comment]
Good point! 

But I think it's not needed, we have no control ever it. Like real app: content can't be translated and the app just got system language.

**Comment by drucelweisse** [comment]
I don't know is this correct approach to skip test if provided locale doesn't match test cases? Also tried to use Locale.current.languageCode but it doesn't override with XCTest Plan

**Comment by akaDuality** [comment]
Test language can be set-upped at scheme settings. We can hardcode English

**Comment by drucelweisse** [comment]
Well, as I understood, I need to replace current A11yDescription tests with English one, correct? This is about duplicates I created before.

**Comment by akaDuality** [comment]
Yes, you can just replace it, there in no need to text Russian version explicitly

File: Document/Tests/DocumentTests/A11yDescriptionTests.swift [hunk 1]
```diff
@@ -8,55 +8,55 @@
...
     }
     
-    func test_selectedButton() {
+    func test_selectedButton() throws {
         let descr = A11yDescription.testMake(
-            label: "Город",
-            value: "Екатеринбург",
+            label: "City",
+            value: "Yekaterinburg",
             trait: [.button, .selected])
```
Comment (drucelweisse) [inline Document/Tests/DocumentTests/A11yDescriptionTests.swift:41]: This test is failing due Selected trait generates "Selected: City: Yekaterinburg. Button." Should it use dot in the place?

File: Document/Tests/DocumentTests/A11yDescriptionTests.swift [hunk 2]
```diff
@@ -8,55 +8,55 @@
...
         
-        XCTAssertEqual(descr.voiceOverText, "Выбрано. Город: Екатеринбург. Кнопка")
+        XCTAssertEqual(descr.voiceOverText, "Selected. City: Yekaterinburg. Button.")
```
Comment (drucelweisse) [inline Document/Tests/DocumentTests/A11yDescriptionTests.swift:42]: This test is failing due Selected trait generates "Selected: City: Yekaterinburg. Button." Should it use dot in the place?

Comment (akaDuality) [reply Document/Tests/DocumentTests/A11yDescriptionTests.swift:42]: I don't know yet or it's not important. As far as I can see VoiceOver doesn't place the dot at the end of a sentence.

When we will create text-based strategy we will have to synchronise our text with the strategy, but we will do it later.

**Review by akaDuality — APPROVED** [review]
